app/src/main/java/com/android/sample/ui/utils/SampleClasses.kt

```diff
@@ -21,11 +22,33 @@ val recipeOrigins = arrayOf("Italy", "India", "United Kingdom", null)
 
 val recipeInstructions =
     arrayOf(
-        "1. Boil water\n2. Add pasta\n3. Cook for 10 minutes\n4. Drain water\n5. Add sauce",
-        "1. Boil water\n2. Add chicken\n3. Cook for 10 minutes\n4. Drain water\n5. Add sauce",
-        "1. Boil water\n2. Add fish\n3. Cook for 10 minutes\n4. Drain water\n5. Add chips",
-        "take it out of the bag and shove it in your mouth",
-    )
+        listOf(
+            Instruction(description = "1. Boil water", time = "30 min", iconType = "Cook"),
+            Instruction(description = "2. Add pasta", time = "30 min", iconType = "Fire"),
+            Instruction(description = "3. Cook for 10 minutes", time = "30 min", iconType = "Fire"),
```

Comment:
Nothing serious, but it made me smile that in the instruction it says to cook for 10 min and that the time tag is 30 min

Reply:
Yeah, I guess the one that wrote these tests must have been really tired